src/static/riot/submissions/submission_management.tag

```diff
@@ -0,0 +1,266 @@
+            </tr>
+            <tr>
+                <td class="collapsing">
+                    <div class="ui checkbox">
+                        <input type="checkbox"> <label></label>
+                    </div>
+                </td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+            </tr>
+            <tr>
+                <td class="collapsing">
+                    <div class="ui checkbox">
+                        <input type="checkbox"> <label></label>
+                    </div>
+                </td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+            </tr>
+            <tr>
+                <td class="collapsing">
+                    <div class="ui checkbox">
+                        <input type="checkbox"> <label></label>
+                    </div>
+                </td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+            </tr>
+            <tr>
+                <td class="collapsing">
+                    <div class="ui checkbox">
+                        <input type="checkbox"> <label></label>
+                    </div>
+                </td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+            </tr>
+            <tr>
+                <td class="collapsing">
+                    <div class="ui checkbox">
+                        <input type="checkbox"> <label></label>
+                    </div>
+                </td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+            </tr>
+            </tbody>
+            <tfoot class="full-width">
+            <tr>
+                <th colspan="11">
+                    <div class="ui right floated pagination menu">
+                        <a class="icon item"><i class="left chevron icon"></i></a>
+                        <a class="item">1</a>
+                        <a class="item">2</a>
+                        <a class="item">3</a>
+                        <a class="item">4</a>
+                        <a class="icon item"><i class="right chevron icon"></i></a>
+                    </div>
+                </th>
+            </tr>
+            </tfoot>
+        </table>
+    </div>
+    <!-- Javascript -->
```

Comment:
Hmm, I like the idea of these, I think we can get the same idea with just space? I think 2 spaces between HTML - JS - CSS should be fine. What do you think?

Also, commented "sections" in JS we'd normally do something like...

```js
        /* ----------------------------------------------------------------------------
         Initializer
         ----------------------------------------------------------------------------*/

... 
        /* ----------------------------------------------------------------------------
         Event handlers
         ----------------------------------------------------------------------------*/

```

Reply:
removed comment